chef/cookbooks/hyperv/recipes/config_ceilometer.rb

```diff
@@ -5,55 +5,13 @@
 bind_host = node[:ceilometer][:api][:host]
 bind_port = node[:ceilometer][:api][:port]
 
-if node[:ceilometer][:use_mongodb]
-  db_connection = nil
-
-  if node[:ceilometer][:ha][:server][:enabled]
-    db_hosts = search(:node,
-                      "ceilometer_ha_mongodb_replica_set_member:true AND roles:ceilometer-server AND "\
-                      "ceilometer_config_environment:#{node[:ceilometer][:config][:environment]}"
-      )
-    unless db_hosts.empty?
-      ## This will fail on windows, we need to find something better here
-      mongodb_servers = db_hosts.map { |s| "#{Chef::Recipe::Barclamp::Inventory.get_network_by_type(s, "admin").address}:#{s[:ceilometer][:mongodb][:port]}" }
-      db_connection = "mongodb://#{mongodb_servers.sort.join(',')}/ceilometer?replicaSet=#{node[:ceilometer][:ha][:mongodb][:replica_set][:name]}"
-    end
-  end
-
-  # if this is a cluster, but the replica set member attribute hasn't
-  # been set on any node (yet), we just fallback to using the first
-  # ceilometer-server node
-  if db_connection.nil?
-    db_hosts = search(:node, "roles:ceilometer-server")
-    db_host = db_hosts.first || node
-    ## This will fail on windows
-    mongodb_ip = Chef::Recipe::Barclamp::Inventory.get_network_by_type(db_host, "admin").address
-    db_connection = "mongodb://#{mongodb_ip}:#{db_host[:ceilometer][:mongodb][:port]}/ceilometer"
-  end
-else
-  db_settings = fetch_database_settings
-
-  include_recipe "database::client"
-  include_recipe "#{db_settings[:backend_name]}::client"
-  include_recipe "#{db_settings[:backend_name]}::python-client"
-
-  db_password = ""
-  # pickup password to database from ceilometer-server node
-  node_controllers = search(:node, "roles:ceilometer-server") || []
-  if node_controllers.length > 0
-    db_password = node_controllers[0][:ceilometer][:db][:password]
-  end
-
-  db_connection = "#{db_settings[:url_scheme]}://#{node[:ceilometer][:db][:user]}:#{db_password}@#{db_settings[:address]}/#{node[:ceilometer][:db][:database]}"
-end
-
-time_to_live = node[:ceilometer][:database][:time_to_live]
-if time_to_live > 0
+metering_time_to_live = node[:ceilometer][:database][:metering_time_to_live]
+if metering_time_to_live > 0
   # We store the value of time to live in days, but config file expects seconds
-  time_to_live = time_to_live * 3600 * 24
+  metering_time_to_live = metering_time_to_live * 3600 * 24
 end
 
-is_compute_agent = %w(ceilometer-agent-hyperv nova-multi-compute-hyperv).all?{ |role| node.roles.include?(role) }
+is_compute_agent = %w(ceilometer-agent-hyperv nova-compute-hyperv).all?{ |role| node.roles.include?(role) }
```

Comment:
Metrics/LineLength: Line is too long. [107/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceBeforeBlockBraces: Space missing to the left of {.